data/human/human missions.txt

```diff
@@ -6890,3 +6890,1716 @@ mission "Hogshead's Dilemma II"
+			`	You cut him off. "It's fine, Timothy. I'll take you to Rand."`
+			`	Timothy breaks down in tears. "Thank you. Thank you. Thank you..." he mumbles over and over again.`
+			`	You help him out of the crate and get him cleaned up.`
+				accept
+
+	on complete
+		event "timothy radrickson: on rand" 340
+		conversation
+			`During the trip Timothy's spirits and health improved, and by the time you arrive on Rand he seems like a different man. Before he steps out of the airlock he turns to you one more time. "Thank you so much, Captain <last>." He shakes your hand."`
+			`	"I mean it. From the bottom of my heart. You've saved my life, and given me hope. I won't forget this, or you. I can never thank you enough."`
+			`	His eyes glistening, Timothy turns from you and exits your ship. Hopefully he finds what he is looking for on Rand.`
+
+
+mission "Timothy Radrickson 1a: Accepted"
+	landing
+	name "Stowaway to <planet>"
+	description "You found Timothy on your ship stowed away in a food crate, on the run from someone. He requests you to bring him to <destination>, where he can find work."
+	source
+		government "Republic" "Free Worlds" "Independent" "Pirate" "Neutral"
+	destination "Rand"
+	to offer
+		has "Timothy Radrickson 1: Stowaway: active"
+	on abort
+		fail "Timothy Radrickson 1: Stowaway"
+
+
+event "timothy radrickson: on rand"
+event "timothy radrickson: mummy"
+
+
+mission "Timothy Radrickson 2: mummy"
+	landing
+	invisible
+	to offer
+		has "Timothy Radrickson 1: Stowaway: failed"
+		has "event: timothy radrickson: mummy"
+	on enter
+
+	on enter
+		conversation
+			`For days there's been an odd lingering scent filtering in from your pressurized cargo; the smell has finally permeated your command deck, and something had to be done. A ship wide search has led you to a unopened foodstuffs crate. Although the crate was unopened, the seal on it appeared tampered with and the conjecture was that the food had gone bad.`
+			`	Upon opening it you discovered that was in fact, not the case. Instead you discovered the decaying body of a dead and forgotten stowaway; the entire crate promptly found its way to your airlock. Despite the diligent work of your ship's air filters, you can't quite rid the ship of the smell.`
+
+
+mission "Timothy Radrickson 2: Interlude"
+	landing
+	to offer
+		has "Timothy Radrickson 1: Stowaway: failed"
+		has "timothy radrickson: port"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	on offer
+		conversation
+			`Blinking at the light Timothy shuffles toward the exit. "Th-th-thank you. Thank you so much," Timothy mumbles.`
+			`	Timothy opens the hatch and looks back at you. "You s-s-saved my life. I won't forget. I won't."`
+			`	With that Timothy exits your ship and begins his new life. You hope everything works out for the best.`
```

Comment:
```suggestion
			`	With that, Timothy exits your ship and begins his new life. You hope everything will work out for the best.`
```

Reply:
good with the added comma, thor knows my comma usage is terrible, but adding 'will' just comes off to me as slightly more wordy for the sake of being slightly more wordy?

Reply:
I agree after looking back at it.